_includes/embed/audio.html

```diff
@@ -0,0 +1,43 @@
+{% assign src = include.src | strip %}
+{% assign title = include.title | strip %}
+{% assign types = include.types | default: '' | strip | split: '|' %}
+
+{% unless src contains '://' %}
+  {%- capture src -%}
+    {% include img-url.html src=src %}
```

Comment:
```suggestion
    {% include img-url.html src=src img_path=page.img_path %}
```

We have no dedicated alternative variable for video and audio at the moment like `img_path` for images. However, please consider to unify behavior between video and audio embedding: use `img_path` for audio, or remove for video. Would be great to have separate dedicated variable for video and audio later. Taking into account the warning in documentation, it may make sense to remove usage of `img_url` for video embedding.